FlexMeasures/flexmeasures
# add quantity or sensor storage scheduler

## Description

So far, the flex-model fields of type `QuantityOrSensor` could only be used trough the API. This PR introduces them in command `flexmeasures add schedule for-storage`.

### How to test?

This PR introduces a new end-to-end test to check if we can trigger the storage schedule via the CLI. It is tagged to run only on a local dev environment, therefore, I would like if the reviewer could try (him/her)self the test locally.

### TODO

- [x] Create and end-to-end test for the command `flexmeasures add schedule for-storage`.
- [x] CLI changelog entry.


## Related Items

Closes https://github.com/FlexMeasures/flexmeasures/issues/935

---
